### ISSUE TITLE
1.12 Gelid Enderium Pickaxe not putting items in bound inventory

### ISSUE THREAD

**Issue by Zygus42** [thundR/RedstoneRepository#9]
RedstoneRepository-1.12.2-1.3.2-36-universal
forge-1.12.2-14.23.4.2703-installer-win

Just made myself a Gelid Enderium Pickaxe, powered it up and used shift+right-click to attach it to a vanilla oak chest, but it's not putting mined items in the bound inventory.  When I use control, it says it's bound to the correct location of the chest.

**Comment by thundr** [comment]
I can't reproduce this. Can you give any more details?

**Comment by Zygus42** [comment]
Did some further testing, and it appears that it works fine until you put an enchantment on the tool.  Tested with:  Holding 4, Efficiency 5 & Silk Touch, all together and one at a time - as soon as an enchantment goes on, no longer works.

**Comment by thundr** [comment]
Can you tell me what the command /cofh hand -yourname- nbt gives you with the nonfunctional pick in your hand?

**Comment by thundr** [comment]
Yeah I still cannot reproduce this. Chances are this is caused by some intermod interaction. Can you put a modlist into paste.ee or ghostbin, or another paste service?

**Comment by Zygus42** [comment]
My last test was with no mods other than the required mods to make it run - cofh core and such.  I guess I can rebuild it again........

**Comment by Zygus42** [comment]
CodeChickenLib-1.12.2-3.2.0.345-universal
CoFHCore-1.12.2-4.5.2.19-universal
CoFHWorld-1.12.2-1.2.0.5-universal
RedstoneArsenal-1.12.2-2.5.1.13-universal
RedstoneFlux-1.12-2.0.2.3-universal
RedstoneRepository-1.12.2-1.3.2-36-universal
ThermalExpansion-1.12.2-5.5.0.29-universal
ThermalFoundation-1.12.2-2.5.0.19-universal
- minimum files needed to make this work.  No other mods installed.

Sequence of events that always fails:
give yourself Gelid Enderium Pickaxe
enchant with Holding 4
enchant with Efficiency 5
enchant with Silk Touch
press v to empower
shift+right-click on chest
use /gamemode 0
go mine

![2018-07-12_08 43 54](https://user-images.githubusercontent.com/20757445/42606478-74088cd6-85b0-11e8-81a1-1a414bc38aca.png)
